test/e2e/apimachinery/aggregator.go

```diff
@@ -95,28 +95,41 @@ var _ = SIGDescribe("Aggregator", func() {
 	})
 })
 
-func cleanTest(ctx context.Context, client clientset.Interface, aggrclient *aggregatorclient.Clientset, namespace string) {
+func cleanupSampleAPIServer(ctx context.Context, client clientset.Interface, aggrclient *aggregatorclient.Clientset, names sampleAPIServerObjectNames) {
```

Comment:
If you call it 'n' instead of names, maybe you won't feel the need to make a local copy of namespace? :)